index.html

```diff
@@ -1,184 +1,187 @@
+
+<!-- updated preferences to auto-convert tabs to spaces per comment: thanks for the tip, that's been a quality issue I've known I would run into. -->
+<!-- indented head and body into html tag. Noted boilerplate DOES NOT DO THIS AUTOMATICALLY, I will need to manually edit for all future docs. -->
 <!-- Validated using html validator -->
 <!DOCTYPE html>
 <html>
-<head>
-	<meta charset="utf-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<title>Megan Ford NSS HTML quiz</title>
-	<link rel="stylesheet" href="quiz.css">
-</head>
-<!-- NOTE: all image sources are 200px wide and square- resized to 150px in html markup. V2 will resize image sources if needed.-->
-<body>
-<div class="main-content"> <!--included in case a full-width header/footer is added later -->
+  <head>
```

Comment:
The title link & 2 meta tags are indented with tabs instead of the 2-space convention you have in the rest of the document